internal/backend/rename_windows.go

```diff
@@ -0,0 +1,54 @@
+package backend
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/alexbrainman/goissue34681"
```

Comment:
This feels wrong lol I wonder if the author will keep that around. I guess that in the worst case it'll be cached in the goproxy and we can fork it.